runners/direct-java/pom.xml

```diff
@@ -70,6 +70,14 @@
             <beamUseDummyRunner>false</beamUseDummyRunner>
           </systemPropertyVariables>
         </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>org.apache.beam</groupId>
+            <artifactId>beam-runners-core-java</artifactId>
+            <type>test-jar</type>
+            <version>${project.version}</version>
```

Comment:
If this version is left out, maven crashes on it, not taking the version from `dependencyManagement`. Yet in `runners-parent` it did not require a version. I don't know why.